ccd-definition/ComplexTypes/CareSupervision/HearingPreferences.json

```diff
@@ -2,150 +2,55 @@
   {
     "LiveFrom": "01/01/2017",
     "ID": "HearingPreferences",
-    "ListElementCode": "hearingPreferencesLabel",
-    "FieldType": "Label",
-    "ElementLabel": "Is any of the following needed to help someone take part in a hearing?",
-    "SecurityClassification": "Public"
-  },
-  {
-    "LiveFrom": "01/01/2017",
-    "ID": "HearingPreferences",
-    "ListElementCode": "interpreter",
-    "FieldType": "YesOrNo",
-    "ElementLabel": "Interpreter",
```

Comment:
I wonder if instead of moving these to the deprecated file (which may cause confusion later as data will be hanging around with this stuff, and then models don't line up 1:1 with the ComplexType files, if it may be better leaving them in and just showConditioning them to something = "DEPRECATED" instead of "DO_NOT_SHOW" (and maybe moving to the bottom of the file?)

In two minds about it, as the DEPRECATED file already exists...